__tests__/app.test.js

```diff
@@ -330,47 +296,137 @@ describe('PATCH/api/articles/:article_id', () => {
 				});
 			});
 	});
-  test('returns 400 if id is not a valid format', () => {
-        const item = {inc_votes: 11};
-        return request(app)
-        .patch('/api/articles/incorretID')
-        .send(item)
-        .expect(400)
-        .then(({body}) => {
-            const {msg} = body;
-            expect(msg).toBe('Invalid ID');
-        })
+	test('returns 400 if id is not a valid format', () => {
+		const item = { inc_votes: 11 };
+		return request(app)
+			.patch('/api/articles/incorretID')
+			.send(item)
+			.expect(400)
+			.then(({ body }) => {
+				const { msg } = body;
+				expect(msg).toBe('Invalid ID');
+			});
+	});
 
-});
+	test('returns a 400 if inc_votes is missing.', () => {
+		const item = {};
+		return request(app)
+			.patch('/api/articles/3')
+			.send(item)
+			.expect(400)
+			.then(({ body }) => {
+				const { msg } = body;
+				expect(msg).toBe('Invalid Format');
+			});
+	});
 
-test ('returns a 400 if inc_votes is missing.', () => {
-        const item = {};
-        return request(app)
-        .patch('/api/articles/3')
-        .send(item)
-        .expect(400)
-        .then(({body}) => {
-            const {msg} = body;
-            expect(msg).toBe('Invalid Format');
-        });
-});
+	test('returns a 400 if votes is not a number.', () => {
+		const item = { inc_votes: 'invalid data type' };
+		return request(app)
+			.post('/api/articles/3/comments')
+			.send(item)
+			.expect(400)
+			.then(({ body }) => expect(body.msg).toBe('Invalid format'));
+	});
 
+	test('returns a 404 if ID is not found.', () => {
+		const item = { inc_votes: 'invalid data type' };
+		return request(app)
+			.post('/api/articles/8989898')
+			.send(item)
+			.expect(404)
+			.then(({ body }) => expect(body.msg).toBe('Incorrect File Path'));
```

Comment:
I would make this error message different to your 'endpoint not found' message. You'll be the client using this BE in a few weeks, and 'incorrect file path' might make it harder to debug if you get this message